.editorconfig

```diff
@@ -8,6 +8,10 @@ indent_size = 2
 insert_final_newline = true
 trim_trailing_whitespace = true
 
+[*.snap]
+max_line_length = off
+trim_trailing_whitespace = false
+
```

Comment:
Note: we could have combined both the `[*.snap]` and `[*.md]` lines into a single statement, left separate in a clarity effort, plus we're only minimally confident this will solve the problem and it'll be easier to remove.